src/js/core.js

```diff
@@ -395,6 +395,9 @@ function MediumEditor(elements, options) {
         AnchorPreview: AnchorPreview
     };
 
+    MediumEditor.Util = Util;
+    MediumEditor.Selection = Selection;
```

Comment:
Yeah, you make some great points and I feel much better about it.

Also, I see no reason why you can't just call them `.util` and `.selection`, I agree that they're better.  Might as well do it now before anyone takes any dependencies on it :wink: